specification/computeschedule/ComputeSchedule.Management/client.tsp

```diff
@@ -96,9 +96,9 @@ using Microsoft.ComputeSchedule;
   "csharp"
 );
 @@clientName(CancelOperationsRequest.correlationid, "correlationId", "csharp");
```

Comment:
do we need to update this one: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/computeschedule/ComputeSchedule.Management/models.tsp#L232-L237?